Core/include/Acts/Detector/detail/ReferenceGenerators.hpp

```diff
@@ -39,12 +39,12 @@ struct CenterReferenceGenerator {
 
 /// A struct to access reference positions based on bin values
 ///
+/// @tparam bVAL the binning value to be used for the binning position call
+///
 /// This generator will provide only one filling point and hence
 /// only a single bin in the indexed grid.
+template <BinningValue bVAL = BinningValue::binValues>
```

Comment:
Does this need to go to the template parameters?

Reply:
It doesn't, it's lets the generator being default constructed, which is very handy.